moncli/types.py

```diff
@@ -77,4 +78,12 @@ def _convert(self, value: tuple):
         return data
 
     def _export(self, value):
-        return value
\ No newline at end of file
+        return value
+
+class TextType(MondayType):
+    native_type = str
+    allow_casts = (int, float)
+    null_value = ""
+
+    def _export(self, value):
```

Comment:
You should not have to define the _export function for this instance.  

When to_primitive is called, you may always assume that the value is either the native default (None, [], etc.) or the native type itself.  And if not, thanks to a merging in of some enhancements, it will convert the data to native before formatting.